includes/Modules/Idea_Hub.php

```diff
@@ -241,7 +241,7 @@ function( $data ) {
 					if (
 						// Do nothing if tracking is disabled or if it is enabled and already allowed.
 						empty( $data['trackingEnabled'] ) ||
-						! empty( $data['trackingAllowed'] ) ||
+						! empty( $data['isSiteKitScreen'] ) ||
```

Comment:
@ivankruchkoff with all changes in this PR, we probably don't need this hook for the `googlesitekit_inline_base_data` filter at all. Let's remove it and verify that events are still tracked correctly on the posts list page.

Reply:
Good point @eugene-manuilov – we actually need to remove this because keeping this would result in pageviews tracked for the post list screen which we don't want. This logic was just a workaround to enable tracking on this page in the first place which will no longer be needed as it will be automatically loaded on-demand. This is not a SK screen though so we want to avoid it being considered as one, and by extension sending a pageview.

Reply:
https://github.com/google/site-kit-wp/pull/4142/commits/4853bb30bda145a095d9d69f15ca0a440115c3d4